sql/changes/1.9/workflow-email-table.sql

```diff
@@ -0,0 +1,21 @@
+
+create table email (
+   workflow_id int primary key references workflow(workflow_id),
+   "from" text,
+   "to" text,
+   "cc" text,
+   "bcc" text,
+   "notify" boolean default false,
+   subject text,
+   body text,
+   sent_date date
+   );
+
+insert into file_class values (8, 'e-mail');
+
+CREATE TABLE file_email (
+       check (file_class=8),
```

Comment:
file_class=8? Neumonics would help lisibility & understanding

Reply:
True. It is an exact copy of the same code in pg-database. It is the same 8 as the one mentioned above which is being inserted into the file_class table. Values 1 through 7have been taken by other attachment types.
Adding a mnemonic here can't be done: the file isn't loaded through psql

Reply:
Thinking about it, I don't know how to resolve this comment. Unless you have any ideas, I'm afraid I'll have to leave it unresolved.